api/HTMLIFrameElement.json

```diff
@@ -1012,55 +1012,6 @@
           }
         }
       },
-      "setNfcFocus": {
-        "__compat": {
-          "mdn_url": "https://developer.mozilla.org/docs/Web/API/HTMLIFrameElement/setNfcFocus",
```

Comment:
I have redirected https://developer.mozilla.org/en-US/docs/Web/API/HTMLIFrameElement/setNfcFocus to https://developer.mozilla.org/en-US/docs/Web/API/HTMLIFrameElement.